packages/ui/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM node:18-alpine AS base
+
+# Install dependencies only when needed
+FROM base AS deps
```

Comment:
is deps needed?

Reply:
not sure, I'm not an expert in Dockerfiles but this was also used in the official next.js example